### PR TITLE
Support expect as a valid identifier too

### PR DESCRIPTION
**Pull request by aryx**
On our 500K corpus of kotlin code, we get from 83.1% to 83.7% of correct parsing.
Not as a big as my previous PR but still amounts to 3500 more LOC parsed correctly.

### CODE REVIEW

**Comment by aryx** [comment]
cc @fwcd

File: grammar.js [hunk 1]
```diff
@@ -974,7 +978,11 @@ module.exports = grammar({
     // Identifiers
     // ==========
 
-    simple_identifier: $ => $._lexical_identifier, // TODO
+    simple_identifier: $ => choice(
+	$._lexical_identifier,
+	"expect",
+	// TODO: far more identifierOrSoftKeyword
```
Comment (fwcd) [inline grammar.js:984]: For consistency, please use 2-space indent instead of tabs

**Review by fwcd — COMMENTED** [review]
Thanks for the PR, this looks mostly good, just a minor comment regarding formatting below!

**Comment by aryx** [comment]
Done. Should I resolve the conflicts by pulling from the changes you made 1h ago?

**Comment by aryx** [comment]
@fwcd I've merged main and resolve the conflicts.

**Pull request merged**